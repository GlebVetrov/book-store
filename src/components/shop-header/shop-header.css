.shop-header {
    border: 3px solid #808080; 
    background-color: #3d3d3d;   
    padding: 10px;
    border-radius: 10px;
    margin-bottom: 1rem;
    display: flex;
    justify-content: space-between;
}

.shop-header .logo {
    font-family: 'Playfair Display', Geor, serif;
    font-size: 2.5rem;
    padding-left: 1rem;
}

.shop-header .shopping-cart {
    align-self: center;
    font-size: 1.3rem;
    padding-right: 1rem;
}

.shop-header .shopping-cart .cart-icon {
    font-size: 2.2rem;
    color: cadetblue;
    margin-right: 10px;
}