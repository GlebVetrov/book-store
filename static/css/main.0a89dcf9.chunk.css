.book-list-item{display:flex}.book-list-item .book-cover{flex-shrink:0;width:140px;margin-right:30px}.book-list-item .book-cover img{max-width:100%}.book-list-item .book-details{display:flex;flex-direction:column;justify-content:space-between;padding-bottom:10px;align-items:flex-start}.book-list-item .book-details .book-title{font-size:1.2rem}.book-list-item .book-details .book-price{font-size:1.4rem}.book-list{list-style:none}.book-select{background-color:#007bff}ul{list-style:none;text-align:center}ul li{font-size:20px;display:inline;margin-right:20px}.ActivePageLink{font-weight:700;text-decoration:underline}@keyframes lds-spin{0%{opacity:1;-webkit-transform:scale(1.1);transform:scale(1.1)}to{opacity:0;-webkit-transform:scale(1);transform:scale(1)}}@-webkit-keyframes lds-spin{0%{opacity:1;-webkit-transform:scale(1.1);transform:scale(1.1)}to{opacity:0;-webkit-transform:scale(1);transform:scale(1)}}.lds-spin{position:relative}.lds-spin div>div{position:absolute;width:40px;height:40px;border-radius:50%;background:#25bdf0;-webkit-animation:lds-spin 1s linear infinite;animation:lds-spin 1s linear infinite}.lds-spin div:first-child>div{left:140px;top:80px;-webkit-animation-delay:-.857142857142857s;animation-delay:-.857142857142857s}.lds-spin>div:first-child{-webkit-transform:rotate(0deg);transform:rotate(0deg);-webkit-transform-origin:160px 100px;transform-origin:160px 100px}.lds-spin div:nth-child(2)>div{left:117.40938812000002px;top:126.90988892000001px;-webkit-animation-delay:-.714285714285714s;animation-delay:-.714285714285714s}.lds-spin>div:nth-child(2){-webkit-transform:rotate(51.42857142857143deg);transform:rotate(51.42857142857143deg);-webkit-transform-origin:137.40938812000002px 146.90988892000001px;transform-origin:137.40938812000002px 146.90988892000001px}.lds-spin div:nth-child(3)>div{left:66.64874396px;top:138.49567472px;-webkit-animation-delay:-.571428571428571s;animation-delay:-.571428571428571s}.lds-spin>div:nth-child(3){-webkit-transform:rotate(102.85714285714286deg);transform:rotate(102.85714285714286deg);-webkit-transform-origin:86.64874396px 158.49567472px;transform-origin:86.64874396px 158.49567472px}.lds-spin div:nth-child(4)>div{left:25.94186792px;top:106.03302434px;-webkit-animation-delay:-.428571428571429s;animation-delay:-.428571428571429s}.lds-spin>div:nth-child(4){-webkit-transform:rotate(154.28571428571428deg);transform:rotate(154.28571428571428deg);-webkit-transform-origin:45.94186791999999px 126.03302434px;transform-origin:45.94186791999999px 126.03302434px}.lds-spin div:nth-child(5)>div{left:25.94186792px;top:53.96697566px;-webkit-animation-delay:-.285714285714286s;animation-delay:-.285714285714286s}.lds-spin>div:nth-child(5){-webkit-transform:rotate(205.71428571428572deg);transform:rotate(205.71428571428572deg);-webkit-transform-origin:45.94186791999999px 73.96697566px;transform-origin:45.94186791999999px 73.96697566px}.lds-spin div:nth-child(6)>div{left:66.64874396px;top:21.504325280000003px;-webkit-animation-delay:-.142857142857143s;animation-delay:-.142857142857143s}.lds-spin>div:nth-child(6){-webkit-transform:rotate(257.14285714285717deg);transform:rotate(257.14285714285717deg);-webkit-transform-origin:86.64874396px 41.50432528px;transform-origin:86.64874396px 41.50432528px}.lds-spin div:nth-child(7)>div{left:117.40938812000002px;top:33.09011108px;-webkit-animation-delay:0s;animation-delay:0s}.lds-spin>div:nth-child(7){-webkit-transform:rotate(308.57142857142856deg);transform:rotate(308.57142857142856deg);-webkit-transform-origin:137.40938812000002px 53.09011108px;transform-origin:137.40938812000002px 53.09011108px}.lds-spin div:nth-child(8)>div{left:140px;top:80px;-webkit-animation-delay:.142857142857143s;animation-delay:.142857142857143s}.lds-spin>div:nth-child(8){-webkit-transform:rotate(1turn);transform:rotate(1turn);-webkit-transform-origin:160px 100px;transform-origin:160px 100px}.lds-spin{width:200px!important;height:200px!important;-webkit-transform:translate(-100px,-100px) scale(1) translate(100px,100px);transform:translate(-100px,-100px) scale(1) translate(100px,100px)}input{padding:5px;border-radius:5px;margin-right:5px}.home-page{text-align:center;background-color:#3d3d3d}.shopping-cart-table .total{text-align:right;font-size:1.3rem;margin-right:10px}.shopping-cart-table button{margin-left:5px;margin-top:5px}.shopping-cart-list{margin-top:2rem;display:flex;flex-wrap:nowrap;flex-direction:row;justify-content:space-around}.shopping-cart-list div{flex-basis:10px;flex-shrink:1;flex-grow:1}.item-exit{opacity:1}.item-exit-active{opacity:0;-webkit-transition:opacity .5s ease-in;transition:opacity .5s ease-in}.shop-header{border:3px solid grey;background-color:#3d3d3d;padding:10px;border-radius:10px;margin-bottom:1rem;display:flex;justify-content:space-between}.shop-header .logo{font-family:Playfair Display,Geor,serif;font-size:2.5rem;padding-left:1rem}.shop-header .shopping-cart{align-self:center;font-size:1.3rem;padding-right:1rem}.shop-header .shopping-cart .cart-icon{font-size:2.2rem;color:#5f9ea0;margin-right:10px}.fade-enter{opacity:.01}.fade-enter.fade-enter-active{opacity:1;-webkit-transition:opacity .3s ease-in;transition:opacity .3s ease-in}.fade-exit{opacity:1}.fade-exit.fade-exit-active{opacity:.01;-webkit-transition:opacity .3s ease-in;transition:opacity .3s ease-in}
/*# sourceMappingURL=main.0a89dcf9.chunk.css.map */