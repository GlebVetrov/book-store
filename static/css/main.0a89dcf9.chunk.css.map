{"version":3,"sources":["book-list-item.css","book-list.css","spinner.css","search-panel.css","home-page.css","shopping-cart-table.css","shop-header.css","app.css"],"names":[],"mappings":"AAAA,gBACI,YACJ,CAEA,4BACI,aAAc,CACd,WAAY,CACZ,iBACJ,CAEA,gCACI,cACJ,CAEA,8BACI,YAAa,CACb,qBAAsB,CACtB,6BAA8B,CAC9B,mBAAoB,CACpB,sBACJ,CAEA,0CACI,gBACJ,CAEA,0CACI,gBACJ,CC5BA,WACI,eACJ,CAEA,aACI,wBACJ,CAEA,GACI,eAAgB,CAChB,iBACJ,CAEA,MACI,cAAe,CACf,cAAe,CACf,iBACJ,CAEA,gBACI,eAAiB,CACjB,yBACJ,CCtBA,oBACI,GACE,SAAU,CACV,4BAAkC,CAClC,oBACF,CACA,GACE,SAAU,CACV,0BAA8B,CAC9B,kBACF,CACF,CACA,4BACE,GACE,SAAU,CACV,4BAAkC,CAClC,oBACF,CACA,GACE,SAAU,CACV,0BAA8B,CAC9B,kBACF,CACF,CACA,UACE,iBACF,CACA,kBACE,iBAAkB,CAClB,UAAW,CACX,WAAY,CACZ,iBAAkB,CAClB,kBAAmB,CACnB,6CAA8C,CAC9C,qCACF,CACA,8BACE,UAAW,CACX,QAAS,CACT,0CAA4C,CAC5C,kCACF,CACA,0BACE,8BAA+B,CAC/B,sBAAuB,CACvB,oCAAqC,CACrC,4BACF,CACA,+BACE,yBAA0B,CAC1B,wBAAyB,CACzB,0CAA4C,CAC5C,kCACF,CACA,2BACE,8CAA+C,CAC/C,sCAAuC,CACvC,kEAAmE,CACnE,0DACF,CACA,+BACE,kBAAmB,CACnB,kBAAmB,CACnB,0CAA4C,CAC5C,kCACF,CACA,2BACE,+CAAgD,CAChD,uCAAwC,CACxC,qDAAsD,CACtD,6CACF,CACA,+BACE,kBAAmB,CACnB,kBAAmB,CACnB,0CAA4C,CAC5C,kCACF,CACA,2BACE,+CAAgD,CAChD,uCAAwC,CACxC,2DAA4D,CAC5D,mDACF,CACA,+BACE,kBAAmB,CACnB,iBAAkB,CAClB,0CAA4C,CAC5C,kCACF,CACA,2BACE,+CAAgD,CAChD,uCAAwC,CACxC,0DAA2D,CAC3D,kDACF,CACA,+BACE,kBAAmB,CACnB,wBAAyB,CACzB,0CAA4C,CAC5C,kCACF,CACA,2BACE,+CAAgD,CAChD,uCAAwC,CACxC,oDAAqD,CACrD,4CACF,CACA,+BACE,yBAA0B,CAC1B,iBAAkB,CAClB,0BAA2B,CAC3B,kBACF,CACA,2BACE,+CAAgD,CAChD,uCAAwC,CACxC,2DAA4D,CAC5D,mDACF,CACA,+BACE,UAAW,CACX,QAAS,CACT,yCAA2C,CAC3C,iCACF,CACA,2BACE,+BAAiC,CACjC,uBAAyB,CACzB,oCAAqC,CACrC,4BACF,CACA,UACE,qBAAuB,CACvB,sBAAwB,CACxB,0EAA6E,CAC7E,kEACF,CCzIF,MACI,WAAY,CACZ,iBAAkB,CAClB,gBACJ,CCJA,WACI,iBAAkB,CAClB,wBACJ,CCHA,4BACI,gBAAiB,CACjB,gBAAiB,CACjB,iBACJ,CAEA,4BACI,eAAgB,CAChB,cACJ,CAEA,oBACI,eAAgB,CAChB,YAAa,CACb,gBAAiB,CACjB,kBAAmB,CACnB,4BACJ,CAEA,wBACI,eAAgB,CAChB,aAAc,CACd,WAEJ,CAEA,WACI,SACF,CAEF,kBACI,SAAU,CACV,sCAAiC,CAAjC,8BACF,CCjCF,aACI,qBAAyB,CACzB,wBAAyB,CACzB,YAAa,CACb,kBAAmB,CACnB,kBAAmB,CACnB,YAAa,CACb,6BACJ,CAEA,mBACI,uCAA4C,CAC5C,gBAAiB,CACjB,iBACJ,CAEA,4BACI,iBAAkB,CAClB,gBAAiB,CACjB,kBACJ,CAEA,uCACI,gBAAiB,CACjB,aAAgB,CAChB,iBACJ,CC1BA,YACE,WACF,CACA,8BACE,SAAU,CACV,sCAAiC,CAAjC,8BACF,CACA,WACE,SACF,CAEA,4BACE,WAAa,CACb,sCAAiC,CAAjC,8BACF","file":"main.0a89dcf9.chunk.css","sourcesContent":[".book-list-item {\r\n    display: flex;\r\n}\r\n\r\n.book-list-item .book-cover {\r\n    flex-shrink: 0;\r\n    width: 140px;\r\n    margin-right: 30px;\r\n}\r\n\r\n.book-list-item .book-cover img {\r\n    max-width: 100%;\r\n}\r\n\r\n.book-list-item .book-details {\r\n    display: flex;\r\n    flex-direction: column;\r\n    justify-content: space-between;\r\n    padding-bottom: 10px;\r\n    align-items: flex-start;\r\n}\r\n\r\n.book-list-item .book-details .book-title {\r\n    font-size: 1.2rem;\r\n}\r\n\r\n.book-list-item .book-details .book-price {\r\n    font-size: 1.4rem;\r\n}\r\n",".book-list {\r\n    list-style: none;\r\n}\r\n\r\n.book-select {\r\n    background-color: #007bff;\r\n}\r\n\r\nul {\r\n    list-style: none;\r\n    text-align: center;\r\n}\r\n\r\nul li {\r\n    font-size: 20px;\r\n    display: inline;\r\n    margin-right: 20px;\r\n}\r\n\r\n.ActivePageLink {    \r\n    font-weight: bold;\r\n    text-decoration: underline;\r\n}","@keyframes lds-spin {\r\n    0% {\r\n      opacity: 1;\r\n      -webkit-transform: scale(1.1, 1.1);\r\n      transform: scale(1.1, 1.1);\r\n    }\r\n    100% {\r\n      opacity: 0;\r\n      -webkit-transform: scale(1, 1);\r\n      transform: scale(1, 1);\r\n    }\r\n  }\r\n  @-webkit-keyframes lds-spin {\r\n    0% {\r\n      opacity: 1;\r\n      -webkit-transform: scale(1.1, 1.1);\r\n      transform: scale(1.1, 1.1);\r\n    }\r\n    100% {\r\n      opacity: 0;\r\n      -webkit-transform: scale(1, 1);\r\n      transform: scale(1, 1);\r\n    }\r\n  }\r\n  .lds-spin {\r\n    position: relative;\r\n  }\r\n  .lds-spin div > div {\r\n    position: absolute;\r\n    width: 40px;\r\n    height: 40px;\r\n    border-radius: 50%;\r\n    background: #25bdf0;\r\n    -webkit-animation: lds-spin 1s linear infinite;\r\n    animation: lds-spin 1s linear infinite;\r\n  }\r\n  .lds-spin div:nth-child(1) > div {\r\n    left: 140px;\r\n    top: 80px;\r\n    -webkit-animation-delay: -0.857142857142857s;\r\n    animation-delay: -0.857142857142857s;\r\n  }\r\n  .lds-spin > div:nth-child(1) {\r\n    -webkit-transform: rotate(0deg);\r\n    transform: rotate(0deg);\r\n    -webkit-transform-origin: 160px 100px;\r\n    transform-origin: 160px 100px;\r\n  }\r\n  .lds-spin div:nth-child(2) > div {\r\n    left: 117.40938812000002px;\r\n    top: 126.90988892000001px;\r\n    -webkit-animation-delay: -0.714285714285714s;\r\n    animation-delay: -0.714285714285714s;\r\n  }\r\n  .lds-spin > div:nth-child(2) {\r\n    -webkit-transform: rotate(51.42857142857143deg);\r\n    transform: rotate(51.42857142857143deg);\r\n    -webkit-transform-origin: 137.40938812000002px 146.90988892000001px;\r\n    transform-origin: 137.40938812000002px 146.90988892000001px;\r\n  }\r\n  .lds-spin div:nth-child(3) > div {\r\n    left: 66.64874396px;\r\n    top: 138.49567472px;\r\n    -webkit-animation-delay: -0.571428571428571s;\r\n    animation-delay: -0.571428571428571s;\r\n  }\r\n  .lds-spin > div:nth-child(3) {\r\n    -webkit-transform: rotate(102.85714285714286deg);\r\n    transform: rotate(102.85714285714286deg);\r\n    -webkit-transform-origin: 86.64874396px 158.49567472px;\r\n    transform-origin: 86.64874396px 158.49567472px;\r\n  }\r\n  .lds-spin div:nth-child(4) > div {\r\n    left: 25.94186792px;\r\n    top: 106.03302434px;\r\n    -webkit-animation-delay: -0.428571428571429s;\r\n    animation-delay: -0.428571428571429s;\r\n  }\r\n  .lds-spin > div:nth-child(4) {\r\n    -webkit-transform: rotate(154.28571428571428deg);\r\n    transform: rotate(154.28571428571428deg);\r\n    -webkit-transform-origin: 45.94186791999999px 126.03302434px;\r\n    transform-origin: 45.94186791999999px 126.03302434px;\r\n  }\r\n  .lds-spin div:nth-child(5) > div {\r\n    left: 25.94186792px;\r\n    top: 53.96697566px;\r\n    -webkit-animation-delay: -0.285714285714286s;\r\n    animation-delay: -0.285714285714286s;\r\n  }\r\n  .lds-spin > div:nth-child(5) {\r\n    -webkit-transform: rotate(205.71428571428572deg);\r\n    transform: rotate(205.71428571428572deg);\r\n    -webkit-transform-origin: 45.94186791999999px 73.96697566px;\r\n    transform-origin: 45.94186791999999px 73.96697566px;\r\n  }\r\n  .lds-spin div:nth-child(6) > div {\r\n    left: 66.64874396px;\r\n    top: 21.504325280000003px;\r\n    -webkit-animation-delay: -0.142857142857143s;\r\n    animation-delay: -0.142857142857143s;\r\n  }\r\n  .lds-spin > div:nth-child(6) {\r\n    -webkit-transform: rotate(257.14285714285717deg);\r\n    transform: rotate(257.14285714285717deg);\r\n    -webkit-transform-origin: 86.64874396px 41.50432528px;\r\n    transform-origin: 86.64874396px 41.50432528px;\r\n  }\r\n  .lds-spin div:nth-child(7) > div {\r\n    left: 117.40938812000002px;\r\n    top: 33.09011108px;\r\n    -webkit-animation-delay: 0s;\r\n    animation-delay: 0s;\r\n  }\r\n  .lds-spin > div:nth-child(7) {\r\n    -webkit-transform: rotate(308.57142857142856deg);\r\n    transform: rotate(308.57142857142856deg);\r\n    -webkit-transform-origin: 137.40938812000002px 53.09011108px;\r\n    transform-origin: 137.40938812000002px 53.09011108px;\r\n  }\r\n  .lds-spin div:nth-child(8) > div {\r\n    left: 140px;\r\n    top: 80px;\r\n    -webkit-animation-delay: 0.142857142857143s;\r\n    animation-delay: 0.142857142857143s;\r\n  }\r\n  .lds-spin > div:nth-child(8) {\r\n    -webkit-transform: rotate(360deg);\r\n    transform: rotate(360deg);\r\n    -webkit-transform-origin: 160px 100px;\r\n    transform-origin: 160px 100px;\r\n  }\r\n  .lds-spin {\r\n    width: 200px !important;\r\n    height: 200px !important;\r\n    -webkit-transform: translate(-100px, -100px) scale(1) translate(100px, 100px);\r\n    transform: translate(-100px, -100px) scale(1) translate(100px, 100px);\r\n  }\r\n  ","input {\r\n    padding: 5px;\r\n    border-radius: 5px;\r\n    margin-right: 5px;\r\n}\r\n",".home-page {\r\n    text-align: center;\r\n    background-color: #3d3d3d;    \r\n}\r\n",".shopping-cart-table .total{\r\n    text-align: right;\r\n    font-size: 1.3rem;\r\n    margin-right: 10px;\r\n}\r\n\r\n.shopping-cart-table button {\r\n    margin-left: 5px;\r\n    margin-top: 5px;\r\n}\r\n\r\n.shopping-cart-list {\r\n    margin-top: 2rem;\r\n    display: flex;\r\n    flex-wrap: nowrap;\r\n    flex-direction: row;\r\n    justify-content: space-around;\r\n}\r\n\r\n.shopping-cart-list div{ \r\n    flex-basis: 10px;    \r\n    flex-shrink: 1;\r\n    flex-grow: 1;\r\n\r\n}\r\n\r\n.item-exit {\r\n    opacity: 1;\r\n  }\r\n\r\n.item-exit-active {\r\n    opacity: 0;\r\n    transition: opacity 500ms ease-in;\r\n  }\r\n  ",".shop-header {\r\n    border: 3px solid #808080; \r\n    background-color: #3d3d3d;   \r\n    padding: 10px;\r\n    border-radius: 10px;\r\n    margin-bottom: 1rem;\r\n    display: flex;\r\n    justify-content: space-between;\r\n}\r\n\r\n.shop-header .logo {\r\n    font-family: 'Playfair Display', Geor, serif;\r\n    font-size: 2.5rem;\r\n    padding-left: 1rem;\r\n}\r\n\r\n.shop-header .shopping-cart {\r\n    align-self: center;\r\n    font-size: 1.3rem;\r\n    padding-right: 1rem;\r\n}\r\n\r\n.shop-header .shopping-cart .cart-icon {\r\n    font-size: 2.2rem;\r\n    color: cadetblue;\r\n    margin-right: 10px;\r\n}",".fade-enter {\r\n  opacity: 0.01;\r\n}\r\n.fade-enter.fade-enter-active {\r\n  opacity: 1;\r\n  transition: opacity 300ms ease-in;\r\n}\r\n.fade-exit {\r\n  opacity: 1;\r\n}\r\n\r\n.fade-exit.fade-exit-active {\r\n  opacity: 0.01;\r\n  transition: opacity 300ms ease-in;\r\n}"]}