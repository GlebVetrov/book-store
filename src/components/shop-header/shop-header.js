import React from 'react';
import './shop-header.css';
import { NavLink } from 'react-router-dom';

const ShopHeader = ({ numItems, total }) => {
    return (
        <header className='shop-header row'>
            <NavLink to='/'>
                <div className='logo text-primary' href='#'>BooksStore</div>
            </NavLink>
            <NavLink to='/cart'>
                <div>
                    <i className='cart-icon fa fa-shopping-cart'/>
                    {numItems} items (${total})
                </div>
            </NavLink>
        </header>
    )
}

export default ShopHeader