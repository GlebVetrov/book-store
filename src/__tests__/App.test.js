import React from 'react';
import renderer from 'react-test-renderer';
import '../components/shop-header/shop-header.css';
import { NavLink } from 'react-router-dom';
import ShopHeader from '../components/shop-header/shop-header';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

const num = 10;
const orderTotal = 1000;
const numItems = 10;
const total = 1000;
test('работа ShopHeader', () => {

  // создаём тестовую версию компонента
  const component = renderer.create(
    <BrowserRouter>
      <Switch>
            <ShopHeader numItems={num} total={orderTotal} >        
            </ShopHeader>
      </Switch>
    </BrowserRouter>
    
    
  );

  // получаем снэпшот (HTML-снимок) компонента для сверки, что вёрстка не испортилась
  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
});
