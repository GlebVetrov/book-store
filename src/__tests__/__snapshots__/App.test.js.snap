// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа ShopHeader 1`] = `
<header
  className="shop-header row"
>
  <a
    aria-current="page"
    className="active"
    href="/"
    onClick={[Function]}
    style={Object {}}
  >
    <div
      className="logo text-primary"
      href="#"
    >
      BooksStore
    </div>
  </a>
  <a
    aria-current={null}
    href="/cart"
    onClick={[Function]}
  >
    <div>
      <i
        className="cart-icon fa fa-shopping-cart"
      />
      10
       items ($
      1000
      )
    </div>
  </a>
</header>
`;
